src/main/java/com/example/rets_api/entity/AnimalPolicyEntity.java

```diff
@@ -0,0 +1,34 @@
+package com.example.rets_api.entity;
+
+import lombok.*;
+
+import javax.persistence.*;
+
+import static com.example.rets_api.resource.Enums.*;
+
+@Entity(name = "animal_policy")
+@Setter
+@Getter
+@AllArgsConstructor
+@NoArgsConstructor
+@ToString
+public class AnimalPolicyEntity {
+
+    @Id
+    @GeneratedValue
+    @Column(name="animal_policy_id")
+    private Long animalPolicyId;
+
+    private Indicator animalsPermitted;
+
+    private String permittedTypes;
+
+    private Long weightLimit;
+
+    private WeightUnit weightUnit;
+
+    @OneToOne
```

Comment:
This should be ManyToOne. Because the same AnimalPolicy can be used for several properties, but one property can have just one AnimalPolicy

Reply:
ah ok sure I will change them

Reply:
done